### ISSUE TITLE
Some buttons are missing an appropriate text for accessibility

### ISSUE THREAD

**Issue by johnnyblasta** [origo-map/origo#1297]
**Is your feature request related to a problem? Please describe.**
Some buttons are missing aria-label with an appropiate text for accessibility.

**Describe the solution you'd like**
Go through the interface and check if the aria-label is set with an appropriate text and add for places they are missing.
